other/exposed-laravel-clockwork.bcheck

```diff
@@ -0,0 +1,26 @@
+metadata:
+    language: v1-beta
+    name: "Exposed Laravel Clockwork Endpoint"
+    description: "A Laravel Clockwork php dev tools was discovered, which exposed cleartext HTTP request, responses, passwords, tokens and some information containing app secrets. "
+    author: "Yasin Yilmaz"
+	tags: "laravel", "clockwork", "php"
```

Comment:
Please can you make the indentation consistent.

Reply:
Indentation issues fixed.